### PR TITLE
SvelteKit release candidate migration: error page

### PR DESCRIPTION
**Pull request by EthanThatOneKid**
Copied over required Svelte code from [`/src/routes/__error.svelte`](https://github.com/EthanThatOneKid/acmcsuf.com/blob/36d15b5cdf373b3c594c773ebf3b3942e54677c6/src/routes/__error.svelte) to [`/src/routes/+error.svelte`](https://github.com/EthanThatOneKid/acmcsuf.com/blob/3d96e185f1b04c87f3b3813a9ee17854876f32ea/src/routes/+error.svelte) (see: <https://kit.svelte.dev/docs/routing#error>).

Migration guide: <https://github.com/sveltejs/kit/discussions/5774#discussioncomment-3293209>

### CODE REVIEW

File: src/routes/+error.svelte [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  import { page } from '$app/stores';
+</script>
+
+<svelte:head>
+  <title>ACM at CSUF / {$page.status || 404}</title>
+</svelte:head>
+
+<section title={$page.error?.message}>
+  <em>404</em>
+  <h1 >Frank can't find where you're going!</h1>
```
Comment (diamondburned) [inline src/routes/+error.svelte]: Format.

**Review by diamondburned — APPROVED** [review]
Nothing much to review here. LGTM.

File: src/routes/+error.svelte (hunk 1 above)
Comment (EthanThatOneKid) [reply src/routes/+error.svelte]: Done.

**Review by diamondburned — APPROVED** [review]

**Comment by EthanThatOneKid** [comment]
Replaced by mega-PR #577.

**Pull request closed without merging**